### DISCUSSION TITLE
Get trainer.callback_metrics for best model

### DISCUSSION THREAD

**Discussion by JackCaster** [Lightning-AI/pytorch-lightning#7886] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I have a model checkpoint callback in place to save the best model during training. I would also like to tag this best model with some of the training metrics, for example validation/training accuracy and loss when the model was produced. 

At the end of the training, I can retrieve those metrics from `Trainer.callback_metrics`, but these are not necessarily the ones related to the best model. I tried to restore the trainer with `Trainer(resume_from_checkpoint='best/model/path')` but, unfortunately, the `Trainer.callback_metrics` dictionary is empty. 

Would you be able to help?

**Comment by JackCaster** [comment]
Gently bump ⚡️Would anyone be able to point me in the right direction?
